### ISSUE TITLE
requests module requirement is missing

### ISSUE THREAD

**Issue by gmt4** [rachmadaniHaryono/we-get#45]
**Describe the bug**

TL;DR Python "requests" module requirement is missing, 

See the patch at the bottom of the issue just to save time:
[0001-we-get-Missing-requests-module-update-README.md-install.patch.txt](https://github.com/rachmadaniHaryono/we-get/files/8310809/0001-we-get-Missing-requests-module-update-README.md-install.patch.txt)

What follows is a somewhat long user report, just in case someone else is stumbling on this, and, it saves its time. In the end **we-get** is working, so there's a happy ending.
 
*Thanks!* 

**To Reproduce**

The next steps detail more or less the steps and actions performed to get we-get working:

- Follow the [README.md](https://github.com/rachmadaniHaryono/we-get/blob/master/README.rst#id2) install instructions: 
  The [setup.py](https://github.com/rachmadaniHaryono/we-get/blob/master/setup.py) is nowhere to be found:

   ~~~
    Installation
    =========
    run from the root folder
    .. code-block:: bash
    $ python3 setup.py install
    python3: can't open file 'setup.py': [Errno 2] No such file or directory
   ~~~

   No problem, follow the standard PEP517 build:
   ~~~
   pip3 install --user build # install the build PEP517
   python3 -m build .        # build from sources
   pip3 install --user dist/we-get.tar.gz
   ~~~

- Run the we-get tool we just installed into $HOME/.local/bin
  ~~~
  $HOME/.local/bin/we-get -s ubuntu
  # error: Cannot find target 'yts'.
  ~~~
 
  But the yts module is there and correctly loaded.
  Go ahead, and have a look at the source code.

- The [we_get/core/we_get.py] hides the exception error when importing modules. So change we_get.py to see the exception:
   ~~~
    diff --git a/we_get/core/we_get.py b/we_get/core/we_get.py
    index 39ede99..3c40eec 100644
    --- a/we_get/core/we_get.py
    +++ b/we_get/core/we_get.py
    @@ -160,7 +160,8 @@ class WGSelect(object):
                 try:
                     run = import_module(path)
                 except ImportError:
    -                msg_error("Cannot find target '%s'." % (target), True)
    +                msg_info("Cannot find target '%s'." % (target))
    +                msg_err_trace(True)
   ~~~

 -   Re-run the we-get tool:
 
 ~~~
   $HOME/.local/bin/we-get -s ubuntu
   # Cannot find target 'yts'....
   # error: <class 'ModuleNotFoundError'>
   # No module named 'requests'
   # <traceback object at 0x7fdf049974c8>
   ~~~

 - Bummer. So add the  missing "requests" to [pyproject.toml](https://github.com/rachmadaniHaryono/we-get/blob/master/pyproject.toml).
 
    ~~~
    diff --git a/pyproject.toml b/pyproject.toml
    index f87247d..87e596a 100644
    --- a/pyproject.toml
    +++ b/pyproject.toml
    @@ -33,6 +33,7 @@ colorama = "^0.4.4"
     docopt = "^0.6.2"
     prompt-toolkit = ">=3.0.5"
     pygments = ">=2.6.1"
    +requests = ">=2.27.1"
    ~~~
 - Then, build and install again from source, to ensure requests module is installed.

- Check again, to see if we-get is working:
 ~~~
  ~/.local/bin/we-get -s ubuntu-20
  # Fetching data from <target> ...
  <target> Ubuntu Server 20.04.1 LTS  Raspberry Pi 3 64-bit [2/0] 

 # That's GREAT!
 ~~~

**Comment by rachmadaniHaryono** [comment]
it should be fixed with 1.1.5

the documentation have to be changed little bit to match installation method below the edit
